src/Entity/Redirect.php

```diff
@@ -65,7 +65,7 @@ class Redirect extends ContentEntityBase {
    */
   public static function generateHash($source_path, array $source_query, $language) {
     $hash = array(
-      'source' => $source_path,
+      'source' => strtolower($source_path),
```

Comment:
Sure that this shouldn't use Unicode::strotolower() ? just to be on the safe side (and to make coder sniffer and so on happy)